accessories/climate.js

```diff
@@ -99,6 +103,10 @@ HomeAssistantClimate.prototype = {
     serviceData.entity_id = this.entity_id;
     serviceData.temperature = value;
 
+    if (getTempUnits(this.data) === 'FAHRENHEIT') {
+      serviceData.temperature = celsiusToFahrenheit(serviceData.temperature);
+    }
+  
```

Comment:
I am not sure how this is really expected to work, especially since I am testing this code out and it is not working. HA will still understand that the thermostat is configured as F so it will say the converted value is too low, i.e. 60 F to C is 16, so HA will say it is too low to set.